templates/zerver/help/pin-a-stream.md

```diff
@@ -44,7 +44,7 @@ which is especially handy if you are subscribed to a large number of streams.
 
 {end_tabs}
 
-### Alternate method
+## Alternate method
```

Comment:
Similarly here, maybe it would be better to clarify that the alternate method applies to both pinning and unpinning?